easybuild/tools/testing.py

```diff
@@ -140,6 +183,21 @@ def session_state():
     }
 
 
+def exclude_env_from_report_add(key):
+    """
+    Exclude key from test report if an environment variables contains key.
+    :param key: environment variable to exclude
+    """
+    _exclude_env_from_report.append(key.upper())
+
+
+def exclude_env_from_report_clear():
```

Comment:
This is only used in the tests, it's fine to twiddle with internal things there, so I wouldn't define a custom function for this, and just play with `_exclude_env_from_report` directly in the tests?

Likewise for `exclude_env_from_report_add`